swap/src/protocol/alice/event_loop.rs

```diff
@@ -79,54 +85,77 @@ impl EventLoopHandle {
 #[allow(missing_debug_implementations)]
 pub struct EventLoop {
     swarm: libp2p::Swarm<Behaviour>,
+    peer_id: PeerId,
+    execution_params: ExecutionParams,
+    bitcoin_wallet: Arc<bitcoin::Wallet>,
+    monero_wallet: Arc<monero::Wallet>,
+    db: Arc<Database>,
+    listen_address: Multiaddr,
+
+    // Amounts agreed upon for swaps currently in the setup execution phase
+    // Note: We can do a setup execution with only one peer at a time.
```

Comment:
❓ Why is the setup execution protocol currently restricted to one peer at the time?
As far as I could see the setup is triggered upon swap request. I multiple peers trigger it wouldn't it work? 
I would like to understand better what is missing to allow concurrent swaps (setup).
The hashmap indicates that only one swap per peer is allowed concurrently.

Reply:
One execution setup per peer at a given time. will correct comment to make it clearer.